cmake/pcl_options.cmake

```diff
@@ -64,3 +64,14 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 option(BUILD_tools "Useful PCL-based command line tools" ON)
 
 option(WITH_DOCS "Build doxygen documentation" OFF)
+
+# set index size
+set(PCL_INDEX_SIZE 32 CACHE STRING "Set index size. Only available from PCL 1.12 or greater. Available options are: 8 16 32 64")
+
+#set whether indices are signed or unsigned
+set(PCL_INDEX_SIGNED true CACHE BOOL "Set whether indices need to be signed or unsigned. Signed by default.")
+if (PCL_INDEX_SIGNED)
+  set(PCL_INDEX_SIGNED "true")
```

Comment:
I have the feeling you should probably define another variable e.g. `PCL_INDEX_SIGNED_STR` to avoid touching the user set `PCL_INDEX_SIGNED`.